web/handlers/v2/genomics/lineage_mutations.py

```diff
@@ -66,7 +78,7 @@ async def _get(self):
             query["query"] = create_nested_mutation_query(
                 lineages=query_pangolin_lineage, mutations=query_mutations
             )
-            # print(query)
+            parse_time_window_to_query(date_range_filter, query["query"])
```

Comment:
The current behavior looks like the date range filter isn't operating. I think the problem might be that we need to change this line to update the query with the new time window being calculated -- something like `query["query"] = parse_time_window...` maybe?

Reply:
Looking into the range filter created here:
- https://github.com/outbreak-info/outbreak.api/blob/0fa9506835b18c99b0168d50baa2f1ec90b0a529/web/handlers/genomics/util.py#L302

When making the request to the endpoint below for test purposes:
- /genomics/lineage-mutations?pangolin_lineage=BA.2&frequency=0&gene=ORF1a,ORF1b,S,ORF7b&min_date=2022-01-12&max_date=2022-10-22

The query created is below. The date range was added `date_collected`.
```
{
    "size": 0,
    "query": {
        "bool": {
            "should": [
                {
                    "bool": {
                        "must": [
                            {
                                "term": {
                                    "pangolin_lineage": "BA.2"
                                }
                            },
                            {
                                "range": {
                                    "date_collected": {
                                        "lte": "2022-10-22",
                                        "gte": "2022-01-12"
                                    }
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "aggs": {
        "mutations": {
            "nested": {
                "path": "mutations"
            },
            "aggs": {
                "mutations": {
                    "terms": {
                        "field": "mutations.mutation",
                        "size": 10000
                    },
                    "aggs": {
                        "genomes": {
                            "reverse_nested": {}
                        }
                    }
                }
            }
        }
    }
}
```

The `query` dict is mutable and passed by reference when calling the function below:
```
parse_time_window_to_query(date_range_filter, query["query"])
```

Does it make sense?